iFixit/react-commerce
# Sentry: Make previews report to dev

- Prevents previews from reporting to prod Sentry
- Prevents previews from reporting to prod analytics urls.

## QA
- Make sure errors in the previews are reported to `react-commerce-dev` Sentry, not the prod Sentry.
  - Idk how to do this. You might have to push a commit to this branch that throws an error somewhere, check that it gets reported properly, then remove the commit from this branch.
- Make sure analytics data for previews is sent to matomo.ubreakit.com instead of minerva.ifixit.com

## Before deploy

Update the prod Vercel dashboard with the env vars that were changed.
